pkg/channelwatcher/channel_watcher.go

```diff
@@ -86,8 +86,8 @@ type ShouldWatchFunc func(ch *v1alpha1.Channel) bool
 // 3. Calls the updateConfig func with the new multi-channel-fanout-config.
 // This is used by dispatchers or receivers to update their configs by watching channels.
 func UpdateConfigWatchHandler(updateConfig swappable.UpdateConfig, shouldWatch ShouldWatchFunc) WatchHandlerFunc {
-	return func(ctx context.Context, c client.Client, _ types.NamespacedName) error {
-		channels, err := listAllChannels(ctx, c, shouldWatch)
+	return func(ctx context.Context, c client.Client, chanNamespacedName types.NamespacedName) error {
```

Comment:
Why the change? `chanNamespacedName` doesn't seem to be used.

Reply:
slipped in during merge conflict resolution. This was the original code which was then modified to "_" after PR comments. I had already modified GCP by then. Will revert. Good catch!